src/main.rs

```diff
@@ -29,12 +30,13 @@ use winit::{
 
 const WIDTH: u32 = GB_SCREEN_WIDTH as u32;
 const HEIGHT: u32 = GB_SCREEN_HEIGHT as u32;
+const MENU_BAR: u32 = MENU_BAR_SIZE as u32;
```

Comment:
why is this still necessary if the image is being rendered on the central panel?

Reply:
The gbpixel need to know the size of the menu bar to compute the size of the image with a good aspect ratio.